src/common/topology.js

```diff
@@ -523,17 +523,17 @@ var TopologyMap = function($http, topology) {
         return d.node_public_key;
       });
 
-    locations.enter()
+    locations
```

Comment:
this should actually be chained to the location.enter() statement above, this will work but it will update all of them each time the function is called with the same value, so it might as well only happen when the circle is created.